vllm/worker/hpu_model_runner.py

```diff
@@ -396,6 +396,8 @@ def _set_attn_bias(self, attn_metadata, batch_size, seq_len, device,
             len_mask_v = len_mask.view(batch_size, 1, seq_len, 1)
             mask = attn_mask.logical_or(len_mask).logical_or(len_mask_v)
             off_value = -3E38  #small number, avoid nan and overflow
+            if dtype == torch.float16:
+                off_value = -63000
```

Comment:
maybe a better way is to get the min value via pytorch torch.finfo API like below:
off_value = torch.finfo(dtype).min
this will make the code easy to maintain.